src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -275,7 +275,14 @@ private Status openDB() {
         if (!db.exists()) {
             return new Status(Status.NOT_FOUND);
         }
-        if (!db.open()) {
+
+        boolean isOpen = false;
+        try {
+            isOpen = db.open();
```

Comment:
We should discuss more about this and make it as a standard for cbl java/android.